### PR TITLE
Add test case for OracleJDBCSchemaHandler

### LINKED ISSUE #7760: Add test case for OracleJDBCSchemaHandler

### PR DESCRIPTION
**Pull request by taojintianxia**
Fixes #7760.

### CODE REVIEW

File: shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/metadata/model/physical/jdbc/handler/impl/OracleJDBCSchemaHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.SQLException;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.when;
+
+
```
Comment (terrymanu) [inline shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/metadata/model/physical/jdbc/handler/impl/OracleJDBCSchemaHandlerTest.java:33]: Please only keep one line here

File: shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/metadata/model/physical/jdbc/handler/impl/OracleJDBCSchemaHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+/**
+ * Test for JDBC schema handler of Oracle.
+ */
```
Comment (terrymanu) [inline shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/metadata/model/physical/jdbc/handler/impl/OracleJDBCSchemaHandlerTest.java:36]: Please do not add java doc for test case

File: shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/metadata/model/physical/jdbc/handler/impl/OracleJDBCSchemaHandlerTest.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    @Mock
+    private Connection connection;
+    
+    @Mock
+    private DatabaseMetaData databaseMetaData;
+    
+    @Test
+    public void assertGetType() {
+        assertThat(new OracleJDBCSchemaHandler().getType(), is("Oracle"));
+    }
```
Comment (terrymanu) [inline shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/metadata/model/physical/jdbc/handler/impl/OracleJDBCSchemaHandlerTest.java:51]: It is unnecessary to assert static value

File: shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/metadata/model/physical/jdbc/handler/impl/OracleJDBCSchemaHandlerTest.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    
+    @Test
+    public void assertGetSchema() throws SQLException {
+        when(connection.getMetaData()).thenReturn(databaseMetaData);
+        when(databaseMetaData.getUserName()).thenReturn(USER_NAME);
+        assertThat(new OracleJDBCSchemaHandler().getSchema(connection), is(USER_NAME.toUpperCase()));
```
Comment (terrymanu) [inline shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/metadata/model/physical/jdbc/handler/impl/OracleJDBCSchemaHandlerTest.java:57]: Please use SPI to initialize OracleJDBCSchemaHandler for better test

**Review by terrymanu — CHANGES_REQUESTED** [review]

**Comment by taojintianxia** [comment]
@terrymanu please review again

**Review by terrymanu — APPROVED** [review]

**Pull request merged**